src/commonTest/kotlin/test.kt

```diff
@@ -0,0 +1,28 @@
+package leapcore
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class CommonTests {
+
+    @Test
+    fun outputIsParsedCorrectly() {
+        val outputHexString = "0x000000000000000000000000000000000000000000000000000000000000000a0000b1ccdb544f603af631525ec406245909ad6e1b60"
```

Comment:
Possible to story it outside as constant as we using it twice? (hwere nad on line 20). Same for first argument passed into Output contructor

Reply:
Good point, but this is already obsolete in newer commits.